src/app/views/shared/dynamic-library-examples/example-components/time-selectors/time-selectors.component.html

```diff
@@ -21,7 +21,12 @@
 </div>
 
 <div *ngIf="itemId === items.TimeSelectorDateAndTime.id">
-  <fhi-date-time [dateTime]="dateTime" (dateTimeSelect)="onDateTimeSelect($event)"></fhi-date-time>
+  <fhi-date-time
+    [dateTime]="dateTime"
+    [maxDate]="maxDateTime"
```

Comment:
Only a detail, but `maxDateTime` and `minDateTime` is a bit confusing here, use `maxDate` and `minDate`